octoprint_printoid/pisupport_notifications.py

```diff
@@ -0,0 +1,58 @@
+from .alerts import Alerts
+
+
+class piSupportNotifications:
+
+	def __init__(self, logger):
+		self._logger = logger
+		self._alerts = Alerts(self._logger)
+		self._layers = []
+
+	def handle_piSupportData(self, settings, event_payload):
+		server_url = settings.get(["server_url"])
+		if not server_url or not server_url.strip():
+			# No FCM server has been defined so do nothing
+			return -1
+
+		tokens = settings.get(["tokens"])
+		if len(tokens) == 0:
+			# No Android devices were registered so skip notification
+			return -2
+
+		# anything bad happening?
+		# payload data: {'raw_value': 0, 'current_undervoltage': False, 'past_undervoltage': False, 'current_overheat': False, 'past_overheat': False, 'current_issue': False, 'past_issue': False}
+		if event_payload["current_undervoltage"] is True or event_payload["current_overheat"] is True:
+			self._logger.info("We got current_undervoltage/current_overheat problems")
```

Comment:
may we can maintain which event it is here, such as using an enum:
```
from enum import Enum
class Color(Enum):
     UNDERVOLTAGE = 1
     OVERHEAT = 2
     NOT_SUPPORTED = -1
```
and return if event is NOT_SUPPORTED